### DISCUSSION TITLE
cannot install with compose

### DISCUSSION THREAD

**Discussion by dubefab** [akvorado/akvorado#492] [Q&A] [answered]
Following [Quick start](https://demo.akvorado.net/docs/intro#quick-start)

user@librenms:/opt/akvorado$ sudo curl -sL https://github.com/akvorado/akvorado/releases/latest/download/docker-compose-quickstart.tar.gz | tar zxvf -
docker-compose.yml
tar: docker-compose.yml: Cannot open: Permission denied
docker-compose-demo.yml
tar: docker-compose-demo.yml: Cannot open: Permission denied
.env
tar: .env: Cannot open: Permission denied
orchestrator/clickhouse/data/docker-entrypoint.sh
tar: orchestrator: Cannot mkdir: Permission denied
tar: orchestrator/clickhouse/data/docker-entrypoint.sh: Cannot open: No such file or directory
akvorado.yaml
tar: akvorado.yaml: Cannot open: Permission denied
tar: Exiting with failure status due to previous errors

**Comment by vincentbernat** [comment, marked as answer]
You need to use sudo tar if you are not root.

**Reply by dubefab** [reply]
Thank you ... it works!
